astropy/utils/data.py

```diff
@@ -169,7 +169,7 @@ def is_url(string):
 
 
 def _requires_fsspec(url):
-    """Does the `url` require the optional ``fsspec`` dependency to open?"""
+    """Does the `url` require the optional ``fsspec`` dependency to open?."""
```

Comment:
Oops, it really shouldn't add a dot after a `?` (or `!`, not sure we have those!)

Bug in ruff?

Reply:
I've opened https://github.com/charliermarsh/ruff/issues/1858.
D401 says the line should be in an imperative tense, so even though that's not (yet) turned on, I've rephrased this one-line docstring to "Whether the `url` requires the optional ``fsspec`` dependency to open."

Reply:
Sounds good.